sql/changes/1.5/open_forms_callers.sql

```diff
@@ -0,0 +1,10 @@
+BEGIN;
+
+ALTER TABLE public.open_forms ALTER COLUMN id SET DEFAULT floor(random()*(1000000))+1;
+ALTER TABLE public.open_forms ADD COLUMN caller1 character varying(100);
+ALTER TABLE public.open_forms ADD COLUMN caller2 character varying(100);
+ALTER TABLE public.open_forms ADD COLUMN caller3 character varying(100);
+ALTER TABLE public.open_forms ADD COLUMN caller4 character varying(100);
+ALTER TABLE public.open_forms ADD COLUMN last_used timestamp without time zone;
```

Comment:
I am also unclear as to why this change.  The records get deleted from the table when the session goes away or when the form is closed, so would this not be better handled by logging if it is for debugging?